hash/neogeo.xml

```diff
@@ -11622,7 +11622,36 @@ license:CC0-1.0
 	</software>
 
 	<software name="gladmort">
-		<description>Gladmort (demo)</description>
+		<description>GladMort (demo 2)</description>
```

Comment:
What `demo 2` is supposed to mean, versioning?

Reply:
It's a new available demo, which title contains "demo 2"
![0000](https://github.com/mamedev/mame/assets/26376777/85e3a701-4f0a-43db-a962-4b7ca3622c4a)

Reply:
It's supposed to be `(demo²)` then.

Reply:
> What `demo 2` is supposed to mean, versioning?

OK. Fair enough.
This means that the Amiga CD32 should also be written with superscript style?
![image](https://github.com/user-attachments/assets/a8919bf4-f796-4460-a0f1-12a4fa7cba1a)

Reply:
No. Unless maybe the MAME Amiga driver author(s) insists on it. Likewise, we don't have Odyssey².

Reply:
It's fine for me, yes, assuming it's changed in `amiga/amiga.cpp` too.